src/views/liste-aides.vue

```diff
@@ -123,6 +136,17 @@ const institutionsGroups = computed(() => {
       ),
     }
   }
+  if (selectedBenefits.value) {
+    return {
+      national: filterByBenefit("national"),
+      region: filterByBenefit("region"),
+      departement: filterByBenefit("departement"),
+      epci: filterByBenefit("epci"),
+      commune: filterByBenefit("commune"),
+      caf: filterByBenefit("caf"),
+      msa: filterByBenefit("msa"),
+    }
+  }
```

Comment:
Suggestion d'alternative plus concise (pas obligatoire)

```suggestion
  if (searchTerms.value) {
    return Object.fromEntries(
      Object.keys(types).map((type) => [type, filterByBenefit(type)])
    )
  }
```